src/System.ObjectModel/ref/System.ObjectModel.cs

```diff
@@ -227,18 +227,18 @@ namespace System.Windows.Input
     public partial interface ICommand
     {
         event System.EventHandler CanExecuteChanged;
-        bool CanExecute(object parameter);
-        void Execute(object parameter);
+        bool CanExecute(object? parameter);
+        void Execute(object? parameter);
     }
 }
 namespace System.Windows.Markup
 {
     [System.AttributeUsageAttribute(System.AttributeTargets.Class | System.AttributeTargets.Enum | System.AttributeTargets.Interface | System.AttributeTargets.Method | System.AttributeTargets.Property | System.AttributeTargets.Struct, AllowMultiple=false, Inherited=true)]
     public sealed partial class ValueSerializerAttribute : System.Attribute
     {
-        public ValueSerializerAttribute(string valueSerializerTypeName) { }
-        public ValueSerializerAttribute(System.Type valueSerializerType) { }
-        public System.Type ValueSerializerType { get { throw null; } }
-        public string ValueSerializerTypeName { get { throw null; } }
+        public ValueSerializerAttribute(string? valueSerializerTypeName) { }
+        public ValueSerializerAttribute(System.Type? valueSerializerType) { }
```

Comment:
I think both ctors should be defined to take non-nullable, even though neither validates its input.

Reply:
Yeah agree, I was thinking similar. But its test cases make me think it might should be nullable:

```csharp
public class ValueSerializerAttributeTests
{
    [Theory]
    [InlineData(null, null)]
    [InlineData("", null)]
    [InlineData("propertyName", null)]
    [InlineData("System.Int32", typeof(int))]
    [InlineData("System.int32", null)]
    public void Ctor_String(string valueSerializerTypeName, Type expectedValueSerializerType)
    {
        var attribute = new ValueSerializerAttribute(valueSerializerTypeName);
        Assert.Equal(valueSerializerTypeName, attribute.ValueSerializerTypeName);
        Assert.Equal(expectedValueSerializerType, attribute.ValueSerializerType);
    }

    [Theory]
    [InlineData(null)]
    [InlineData(typeof(int))]
    public void Ctor_Type(Type valueSerializerType)
    {
        var attribute = new ValueSerializerAttribute(valueSerializerType);
        Assert.Equal(valueSerializerType?.AssemblyQualifiedName, attribute.ValueSerializerTypeName);
        Assert.Equal(valueSerializerType, attribute.ValueSerializerType);
    }
}
```
Do you think it doesn't matter?

Reply:
As noted offline, these tests were added as part of a methodical effort to increase code coverage across a large swath of surface area.  The fact that these tests are here checking null doesn't speak to the actual intended usage of the APIs.